src/main/g8/default.properties

```diff
@@ -1,2 +1,5 @@
 name=My Something Project
 description=Say something about this template.
+package=com.example
+scala-version=2.13.6
+scalatest-version=3.2.9
```

Comment:
and this can be bumped to whatever latest 3.2.x is

Reply:
Done